boost-maven/boost-runtimes/runtime-openliberty/pom.xml

```diff
@@ -7,5 +7,76 @@
 	<groupId>boost.runtimes</groupId>
 	<artifactId>openliberty</artifactId>
 	<version>0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+     <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+			<groupId>boost</groupId>
+			<artifactId>boost-common</artifactId>
+			<version>0.1.3-SNAPSHOT</version>
+		</dependency>
+        <dependency>
+            <groupId>boost</groupId>
+            <artifactId>boost-maven-plugin</artifactId>
+            <version>0.1.3-SNAPSHOT</version>
+        </dependency>
+		<dependency>
+			<groupId>net.wasdev.wlp.maven.plugins</groupId>
+			<artifactId>liberty-maven-plugin</artifactId>
+			<version>2.6.3</version>
+		</dependency>
+        <dependency>
+			<groupId>org.apache.maven</groupId>
+			<artifactId>maven-plugin-api</artifactId>
+			<version>2.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.maven.plugin-tools</groupId>
+			<artifactId>maven-plugin-annotations</artifactId>
+			<version>3.4</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
```

Comment:
Since we are adding these dependencies over in the runtime pom, we probably want to remove them from the boost-maven-plugin pom.